### PR TITLE
Update charter.html to better define privacy

### PR DESCRIPTION
**Pull request by bslassey**
fixes issue #6

### CODE REVIEW

**Comment by eriktaubeneck** [comment]
I believe this is mostly aligned with the language discussed earlier today. The only other callout I would make is if we want to use `cross-site/app` and `same-site/app`, instead of only `cross-site` and `same-site`.

**Comment by bslassey** [comment]
Would cross-party and same-party work better?

**Comment by martinthomson** [comment]
I don't think that substituting "party" for "site" would be a good idea.

File: charter.html [hunk 1]
```diff
@@ -163,13 +163,16 @@ <h2>Scope</h2>
             The Working Group will specify new web platform features intended to
             be implemented in browsers or similar user agents. The purpose of
             these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+            privacy guarantees with a strong technical basis that do not enable
```
Comment (martinthomson) [inline charter.html:166]:
```suggestion
            privacy guarantees with a strong technical basis. To maintain privacy, new features will not enable
```

**Review by martinthomson — APPROVED** [review]
This seems fine.  A run-on sentence might be improved, but I think we have the high-level definition we need.

This is not all of what "privacy" might mean in this context, but this is probably a worthwhile contribution nonetheless.  Ultimately, we'll need to define what we really mean through the documentation of our privacy principles (or guidelines) and this cannot capture all that nuance.

As long as we accept that this is not a comprehensive definition, this is fine.

**Comment by eriktaubeneck** [comment]
Agreed with @martinthomson that we shouldn't use "party". I'm also fine with simply using "site", though I'm curious if there's been any discussion of this distinction at the TAG level (cc @darobin).

**Comment by AramZS** [comment]
(Changed the title to make this PR a little more legible.)

**Comment by bmayd** [comment]
Sorry for the late reply, but I find the language somewhat restrictive and would like to propose that:

> The purpose of these features is to support web advertising and provide users with privacy guarantees with a strong technical basis that do not enable cross-site recognition of users as well as not allowing for the same-site recognition of users across the clearing of state or data.

be changed to:

>The purpose of these features is to support advertising while providing users with clearly articulated privacy guarantees with a strong technical basis.

I think this group ought to consider general advertising use-cases supported by online activity to be in scope and not just web advertising.

I also think the focus on cross-site/same-site is too proscriptive for the charter and that we should either leave it out or generalize the language to: "...that do not facilitate unsanctioned cross-context recognition of users."

**Comment by npdoty** [comment]
> As long as we accept that this is not a comprehensive definition, this is fine.

If this is not intended to be a comprehensive definition, then we should say so explicitly. For example, we could precede these examples with "For example,". 

I don't see the need for having these two examples at all, but if they're not explicitly marked as such, then I expect many will be confused as to whether there are other privacy properties of interest to the scope of the Working Group.

**Comment by AramZS** [comment]
Would this group find it objectionable to change this to refer to understanding privacy as defined in the document on Privacy Purposes that the CG will produce. This will also assure that, should the W3C produce a privacy-defining document (as seems to be in progress) we will not have to make a charter change to be aligned with it, but instead a CG discussion. 

Thoughts?

File: charter.html [hunk 2]
```diff
@@ -163,13 +163,16 @@ <h2>Scope</h2>
             The Working Group will specify new web platform features intended to
             be implemented in browsers or similar user agents. The purpose of
             these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+            privacy guarantees with a strong technical basis. To maintain privacy, new features will not enable
```
Comment (benjaminsavage) [inline charter.html:166]: Incorporating @npdoty's feedback, I would suggest we instead say:

The purpose of these features is to support web advertising in private ways. Here "private" refers to appropriate processing of personal information. Examples of ways in which new web platform features might enable inappropriate processing include (but are not limited to):
- enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-cross-site">cross-site recognition of users</a>
- enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-same-site">same-site recognition</a> of users across the clearing of state or data.

Comment (npdoty) [reply charter.html:166]: +1 for explicitly marking these as examples. "appropriate processing of personal information" as a shorthand definition for private is necessarily still incomplete and imperfect, but certainly not the worst definition I've heard. :)

Comment (AramZS) [reply charter.html:166]: @bslassey Do you object to making the above change?

Comment (bslassey) [reply charter.html:166]: No objection, I think that's a great articulation of what we're trying to get across

**Comment by eligrey** [comment]
Are user autonomy losses part of 'privacy'? Curious as to how that aligns here.

Most standards that I expect to be proposed in this group will be exchanging minor amounts of user autonomy for increased value of tracking on the tracker end. This increases the value of your behavior, which appears privacy-adjacent.

File: charter.html (hunk 2 above)
Comment (AramZS) [reply charter.html:166]: Changing `site` to `context` would make this acceptable under consensus.

**Comment by martinthomson** [comment]
I just opened #23, which I hope leaves enough space for us to consider other things, like degradation in autonomy.  Given that given the entire point of advertising sits in a place that is somewhat uncomfortable when you talk about autonomy, I don't want to get *too* far into that discussion, especially in the charter.  What is probably best in this context is to let those aspects of the problem fold into discussions.

**Pull request closed without merging**